src/requestor/outbound-event-serializer.ts

```diff
@@ -6,13 +6,13 @@ export class OutboundEventSerializer implements Serializer {
     if (!value.data) {
       return value;
     }
-    const { options, ...payload } = value.data;
+    const { options, meta, ...payload } = value.data;
 
     return {
       meta: {
         ...(value.meta || {}),
```

Comment:
Guess we could just remove this line to get rid of the meta added by the nestjs in case in the future they changed this part of the logic? So that we could just maintain the customized meta generated by oursevels.

Reply:
good catch, I forget to update it.
I think we can just expand the `meta` here.
```
meta: {
  ... (meta || {}),
  transationId:   meta?.transactionId,
  ...
}
```

Reply:
Agree. ^